src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/AddressDtoValidatorTest.java

```diff
@@ -120,13 +117,37 @@ void testErrorReportedWhenServiceAddressIsNull() {
         String validationMessage = String.format(ValidationMessages.NOT_NULL_ERROR_MESSAGE, EntityDto.SERVICE_ADDRESS_FIELD);
         assertError(parentField, "", validationMessage, errors);
     }
+    @Test
+    void testErrorReportedWhenFictionalCountryIsNotTheListOfAllCountries() {
+        String input = "Utopia";
+        addressDto.setCountry(input);
+        String parentField = EntityDto.PRINCIPAL_ADDRESS_FIELD;
+        Errors errors = addressDtoValidator.validate(parentField, addressDto, CountryLists.getAllCountries(), new Errors(), LOGGING_CONTEXT);
+
+        String validationMessage = String.format(ValidationMessages.COUNTRY_NOT_ON_LIST_ERROR_MESSAGE, input);
+        assertError(parentField, AddressDto.COUNTRY_FIELD, validationMessage, errors);
+    }
 
-    void testErrorReportedWhenFictionalCountryIsNotTheListOfAllCountries(String input) {
-        addressDto.setCountry("Utopia");
+    @Test
+    void testErrorReportedWithoutUnSanitisedStringWhenUnsanitizedCountryIsInput() {
```

Comment:
I'm a bit confused by these two test methods. They both seem to have the same set-up fixture and both call 'validate', so won't the outcome be the same in both methods? I think perhaps the naming of them could be improved - happy to have a call if you wish.

Reply:
Let me try to make the names clearer and if they are not after that then yes we should have a call.

Reply:
Ok, now I follow what you're doing. In theory, you could just have one test method with both/more assertions (as fixture is the same in each) but no harm with it being more verbose.

Thanks for changing those method names.